pkgs/games/bugdom2/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bugdom2";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-+Snw5B5DPhzLPxKu9cThO/CNwoffNkSCLat1h1xRdAM=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+  ];
+  buildInputs = [
+    SDL2
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p "$out/bin"
+    mkdir -p "$out/share/Bugdom2"
+    mv Data ReadMe.txt "$out/share/Bugdom2/"
```

```suggestion
    install -Dm644 Data ReadMe.txt -t $out/share/Bugdom2/
```

Is ReadMe.txt needed by the running program?